SimG4CMS/Calo/src/HGCMouseBite.cc

```diff
@@ -0,0 +1,53 @@
+#include "SimG4CMS/Calo/interface/HGCMouseBite.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
+#include <iostream>
+
+//#define EDM_ML_DEBUG
+
+HGCMouseBite::HGCMouseBite(const HGCalDDDConstants& hgc,
+			   const std::vector<double>& angle, double maxL, 
+			   bool rot) : hgcons_(hgc), cut_(maxL), rot_(rot) {
+
+  for (unsigned int k=0; k<angle.size(); ++k) {
```

Comment:
@bsunanda - this line is equivalent to:
```
for( auto k : angle ) {
```
please, see http://en.cppreference.com/w/cpp/language/range-for